aggregator/entry.go

```diff
@@ -222,12 +242,23 @@ func (e *Entry) AddUntimed(
 	}
 }
 
+// AddTimed adds a timed metric alongside its metadata.
+func (e *Entry) AddTimed(
+	metric aggregated.Metric,
+	metadata metadata.TimedMetadata,
+) error {
+	if err := e.applyValueRateLimit(1, e.metrics.timed.rateLimit); err != nil {
```

Comment:
are there no additional considerations for timer metrics?

Reply:
Nope